db/view/view.cc

```diff
@@ -1962,6 +1993,24 @@ get_view_natural_endpoint(
                 orig_base_endpoints | std::views::transform(std::mem_fn(&locator::node::host_id)),
                 orig_view_endpoints | std::views::transform(std::mem_fn(&locator::node::host_id)));
         return {};
+    } else if (base_endpoints.size() < view_endpoints.size()) {
+        // There are fewer base replicas than view replicas.
+        // This can happen as a result of an RF change when the view replica finishes streaming
+        // before the base replica.
+        // Because of this, a view replica might not get paired with any base replica, so we need
+        // to send an additional update to it.
+        ++cf_stats.total_view_updates_due_to_replica_count_mismatch;
+        no_pairing_replica = view_endpoints.back();
+        if (base_endpoints.size() < view_endpoints.size() - 1) {
```

Comment:
@Deexie will that work when changing RF from 0 -> N incrementally?
Do we do the increments of 1 replica at a time on all tables in the keyspace including base and views?